tools/configure_for_migration_analytics/tasks/set-control-policy.yml

```diff
@@ -0,0 +1,60 @@
+---
+- set_fact:
+    vmware_provider_ids: []
+    provider_type: 'ManageIQ::Providers::Vmware::InfraManager'
+
+- name: Generate an API token
+  shell: |
+    set -o pipefail                    \
+    source {{ miq_profile }} &&        \
+    {{ miq_home_dir }}/bin/rails r "   \
+      puts Api::UserTokenService.new.generate_token('admin', 'api')"
+  register: shell_output
+
+- set_fact:
+    api_token: "{{ shell_output.stdout }}"
+
+- name: CFME | Add Control Profile | Get the profile ID
```

Comment:
There is an ansible role for talking to the manageiq API...would that be preferable to use?